internal/action/actions.go

```diff
@@ -1688,6 +1698,29 @@ func (h *BufPane) HalfPageDown() bool {
 	return true
 }
 
+// PageUpAndMoveCursor scrolls the view up a page and moves the cursor to keep it in view
+func (h *BufPane) PageUpAndMoveCursor() bool {
```

Comment:
While `MoveCursorToView` looks quite generic and useful, these 2 actions `PageUpAndMoveCursor` and `PageDownAndMoveCursor` don't look nice. It should be possible to use chained actions `PageUp,MoveCursorToView` and `PageDown,MoveCursorToView` instead, that's what chained actions are for.

Reply:
You're correct, somehow I missed the part about chaining actions in the docs. Both `Page...AndMoveCursor` actions have been removed.